ReactiveCocoa/Swift/Atomic.swift

```diff
@@ -18,35 +18,40 @@ public final class Atomic<Value> {
 		get {
 			return withValue { $0 }
 		}
-	
 		set(newValue) {
 			modify { _ in newValue }
 		}
 	}
 	
 	/// Initializes the variable with the given initial value.
+	/// - parameter value: Value to be saved by the initialized atomical storage.
```

Comment:
This doesn't match the style of the RAC documentation and doesn't really add any detail. I think it could be removed.